biz.aQute.bnd/test/aQute/bnd/main/TestBndMain.java

```diff
@@ -218,7 +259,7 @@ public void testBuild() throws Exception {
 
 		capturedStdIO.getSystemErrContent()
 			.contains(
-			"000: p: The JAR is empty: The instructions for the JAR named p did not cause any content to be included, this is likely wrong");
+			EMPTY_JAR_MSG);
```

Comment:
in general we can do `assertTrue(processor.check("The JAR is empty:"));` The given strings in the parameters are filtered from the errors and warnings. If anything remains or a string is not found false is returned.

Reply:
We also need to get rid of test which assert that out/err is empty. When run on Java 9+, we get can get warnings from the JVM about illegal access issue which have nothing to do with the test but the test fails anyway.